### ISSUE TITLE
0.13.3 broke builds

### ISSUE THREAD

**Issue by ofek** [pawamoy/mkdocstrings#154]
Hello again!

**Describe the bug**

https://github.com/DataDog/integrations-core/pull/7660/checks?check_run_id=1164408408

```
ERROR   -  Error reading page 'base/api.md': get_handler() got an unexpected keyword argument 'rendering' 
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.5/x64/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/opt/hostedtoolcache/Python/3.8.5/x64/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/runner/work/integrations-core/integrations-core/.tox/docs/lib/python3.8/site-packages/mkdocs/__main__.py", line 192, in <module>
    cli()
  File "/home/runner/work/integrations-core/integrations-core/.tox/docs/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/runner/work/integrations-core/integrations-core/.tox/docs/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/runner/work/integrations-core/integrations-core/.tox/docs/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/runner/work/integrations-core/integrations-core/.tox/docs/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/runner/work/integrations-core/integrations-core/.tox/docs/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/runner/work/integrations-core/integrations-core/.tox/docs/lib/python3.8/site-packages/mkdocs/__main__.py", line 152, in build_command
    build.build(config.load_config(**kwargs), dirty=not clean)
  File "/home/runner/work/integrations-core/integrations-core/.tox/docs/lib/python3.8/site-packages/mkdocs/commands/build.py", line 271, in build
    _populate_page(file.page, config, files, dirty)
  File "/home/runner/work/integrations-core/integrations-core/.tox/docs/lib/python3.8/site-packages/mkdocs/commands/build.py", line 171, in _populate_page
    page.render(config, files)
  File "/home/runner/work/integrations-core/integrations-core/.tox/docs/lib/python3.8/site-packages/mkdocs/structure/pages.py", line 175, in render
    self.content = md.convert(self.markdown)
  File "/home/runner/work/integrations-core/integrations-core/.tox/docs/lib/python3.8/site-packages/markdown/core.py", line 263, in convert
    root = self.parser.parseDocument(self.lines).getroot()
  File "/home/runner/work/integrations-core/integrations-core/.tox/docs/lib/python3.8/site-packages/markdown/blockparser.py", line 90, in parseDocument
    self.parseChunk(self.root, '\n'.join(lines))
  File "/home/runner/work/integrations-core/integrations-core/.tox/docs/lib/python3.8/site-packages/markdown/blockparser.py", line 105, in parseChunk
    self.parseBlocks(parent, text.split('\n\n'))
  File "/home/runner/work/integrations-core/integrations-core/.tox/docs/lib/python3.8/site-packages/markdown/blockparser.py", line 123, in parseBlocks
    if processor.run(parent, blocks) is not False:
  File "/home/runner/work/integrations-core/integrations-core/.tox/docs/lib/python3.8/site-packages/mkdocstrings/extension.py", line 155, in run
    xml_element = self.process_block(identifier, str(block))
  File "/home/runner/work/integrations-core/integrations-core/.tox/docs/lib/python3.8/site-packages/mkdocstrings/extension.py", line 183, in process_block
    handler = get_handler(
  File "/home/runner/work/integrations-core/integrations-core/.tox/docs/lib/python3.8/site-packages/mkdocstrings/handlers/__init__.py", line 244, in get_handler
    handlers_cache[name] = module.get_handler(theme, custom_templates, **config)  # type: ignore
TypeError: get_handler() got an unexpected keyword argument 'rendering'
```

**Additional context**

Did we do something incorrectly?

**Comment by florimondmanca** [comment]
Looks like this is due to `python.py:get_handler()` not accepting `**kwargs` anymore: https://github.com/pawamoy/mkdocstrings/blob/8371e9456278d872f52cbc0b6d4f6d9d81055a8b/src/mkdocstrings/handlers/python.py#L248-L252 :-)

Introduced in this commit: https://github.com/pawamoy/mkdocstrings/commit/25373148699daad42b319fa04213d405ca8a7c4c

**Comment by pawamoy** [comment]
Woops :cold_sweat: 

Looks like I broke it indeed! My apologies!

**Comment by pawamoy** [comment]
Just released 0.13.4 that fixes the issue. Thank you @ofek and @florimondmanca, and sorry again for the broken builds!

**Comment by florimondmanca** [comment]
Thanks for promptly jumping in and releasing a new version, @pawamoy! :-)
